src/erlcloud_sns.erl

```diff
@@ -144,13 +146,13 @@ create_platform_endpoint(PlatformApplicationArn, Token, CustomUserData, Attribut
     erlcloud_xml:get_text(
         "CreatePlatformEndpointResult/EndpointArn", Doc).
 
--spec create_platform_endpoint/6 :: (string(), string(), string(), [{sns_endpoint_attribute(), string()}], string(), string()) -> string().
+-spec create_platform_endpoint/6 :: (parameter(), parameter(), parameter(), [{sns_endpoint_attribute(), parameter()}], parameter(), parameter()) -> string().
```

Comment:
Line 149 is too long: -spec create_platform_endpoint/6 :: (parameter(), parameter(), parameter(), [{sns_endpoint_attribute(), parameter()}], parameter(), parameter()) -> string()..